cldfbench_nominalperson_cldf.py

```diff
@@ -20,63 +20,69 @@ def cmd_download(self, args):
 
         >>> self.raw_dir.download(url, fname)
         """
-        pass
+        wals_dir = self.raw_dir / 'wals'
+        if not wals_dir.exists():
+            wals_dir.mkdir()
+        rec = Record.from_doi('10.5281/zenodo.13950591')
+        rec.download_dataset(wals_dir)
 
     def cmd_makecldf(self, args):
         # Extend the schema:
-        args.writer.cldf.add_component('LanguageTable')
-        args.writer.cldf.add_columns('LanguageTable','Walscode')    # add non-standard column
-        
-        args.writer.cldf.add_component('CodeTable')
+        args.writer.cldf.add_component(
+            'LanguageTable',
+            'Walscode',
+            {'name': 'ISOcodes', 'separator': ';', 'datatype': {'base': 'string', 'format': '[a-z]{3}'}})
```

Comment:
We add a `ISOcodes` column (because CLDF's `ISO639P3code` column expects only a single code), specifying it as list-valued, with each item satisfying the regex `[a-z]{3}` (using properties as described here: https://w3c.github.io/csvw/metadata/#columns).